src/liquid/components/ld-bg-cells/readme.md

```diff
@@ -240,8 +293,42 @@ A background pattern with the Merck cells as additional visual element.
+  </svg>
+</div>
+
+{% endexample %}
+
+## Animated
+
+Adds subtle animations to the background cell, to introduce some dynamics to the brand. Please use animations with caution to avoid user distraction.  
+It is also recommended to only use the property for three layer background cells.
+
+<ld-notice mode="info">
+  Animations will be disabled if user requested that the system minimize the amount of non-essential motion. See <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion">prefers-reduced-motion</a> for more information.
```

Comment:
```suggestion
  Animations will be disabled if the user has set their system preferences to minimize non-essential motion. See <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion">prefers-reduced-motion</a> for more information.
```

Reply:
Tried to phrase it similar to the mdn docs but i am fine adopting your suggestion.